brave/brave-core
# Use only filecoin mainnet by default

Use only filecoin mainnet by default
Resoves https://github.com/brave/brave-browser/issues/23367
Put filecoin testnet under feature flag, mainnet is by default

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Create new wallet
2. Check that there is no filecoin testnet or localhost in network list.
3. Add filecoin acc with different ways(via create, import and hardware import).
4. Check that all dialogs don't contain network chooser and will create filenet account.
5. Check that all accounts starts with 'f'.
6. Make filecoin transaction to created accounts and then back from account with mainnet filecoins.
7. There may be artifacts when update brave with existed filecoin accs, but it is ok because real users don't have filecoin accs yet.
